### ISSUE TITLE
Snap installed fails with "connection to the server 127.0.0.1:16443 was refused"

### ISSUE THREAD

**Issue by mthaddon** [ubuntu/microk8s#1707]
I just tried to install microk8s on an Ubuntu 20.04 instance, and it failed as follows:

```
$ sudo snap install microk8s --classic
error: cannot perform the following tasks:
- Run configure hook of "microk8s" snap if present (run hook "configure":
-----
[...]
+ /snap/microk8s/1769/kubectl --kubeconfig=/var/snap/microk8s/1769/credentials/client.config apply -f /var/snap/microk8s/1769/args/cni-network/cni.yaml
The connection to the server 127.0.0.1:16443 was refused - did you specify the right host or port?
-----)
```

Unfortunately I can't run `microk8s inspect` as it's not installed yet :)

**Comment by ktsakalozos** [comment]
Hi @mthaddon 

Could you try installing the latest/edge channel?
```
sudo snap install micro8s --classic --channel=latest/edge
```

**Comment by mthaddon** [comment]
That seemed to work, thanks.

**Comment by mthaddon** [comment]
This is still failing for me using the stable channel, so I don't think we should consider the issue resolved yet.

**Comment by balchua** [comment]
Reopening until available in `1.19/stable` channel

**Comment by balchua** [comment]
@mthaddon the `1.19/stable` has been updated with the fixes from `latest/edge` if u get the chance to try it. Thanks.
